air/Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
 name = "winter-air"
-version = "0.4.2"
+version = "0.5.0"
 description = "AIR components for the Winterfell STARK prover/verifier"
 authors = ["winterfell contributors"]
 readme = "README.md"
 license = "MIT"
 repository = "https://github.com/novifinancial/winterfell"
-documentation = "https://docs.rs/winter-air/0.4.2"
+documentation = "https://docs.rs/winter-air/0.5.0"
 categories = ["cryptography", "no-std"]
 keywords = ["crypto", "arithmetization", "air"]
 edition = "2021"
-rust-version = "1.60"
+rust-version = "1.67"
```

Comment:
Curious about the toolchain change directly up to 1.67. I assumed there was a breaking change but I'm on 1.66 locally and everything runs smoothly.

Reply:
There was no real need to push it that far (and I think 1.60 would have also worked fine). I do notice that there are minor performance improvements with each new version of Rust - so, didn't want it to fall too far behind. I can change it to 1.66 if that works better though.

Reply:
No need to downgrade it to 1.66 🙂 I was just curious about the rationale behind the bump.